src/pages/RepoPage/ComponentsTab/subroute/ComponentsTable/ComponentsTable.tsx

```diff
@@ -35,6 +35,50 @@ interface ComponentsTableHelper {
   delete: React.ReactElement | null
 }
 
```

Comment:
This adds the table headers (without the actual data) when in the state of either not configured, activate measurements, or is backfilling.